hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/security/OMRenewDelegationTokenRequest.java

```diff
@@ -56,13 +61,31 @@ public OMRenewDelegationTokenRequest(OMRequest omRequest) {
 
   @Override
   public OMRequest preExecute(OzoneManager ozoneManager) throws IOException {
+    // We need to populate user info in our request object.
+    OMRequest request = super.preExecute(ozoneManager);
+
     RenewDelegationTokenRequestProto renewDelegationTokenRequest =
-        getOmRequest().getRenewDelegationTokenRequest();
+        request.getRenewDelegationTokenRequest();
+
+    AuditLogger auditLogger = ozoneManager.getAuditLogger();
+    Map<String, String> auditMap = null;
 
-    // Call OM to renew token
-    long renewTime = ozoneManager.renewDelegationToken(
-        OMPBHelper.convertToDelegationToken(
-            renewDelegationTokenRequest.getToken()));
+    long renewTime;
+    try {
+      Token<OzoneTokenIdentifier> token = OMPBHelper.convertToDelegationToken(
+          renewDelegationTokenRequest.getToken());
+      auditMap = buildTokenAuditMap(token);
```

Comment:
I think we should include renewer in the log in case of failure, too:

```
INFO  | OMAudit | user=testuser/scm@EXAMPLE.COM | ip=172.22.0.4 | op=RENEW_DELEGATION_TOKEN {kind=OzoneToken, service=172.22.0.9:9862, renewer=testuser} | ret=SUCCESS |
ERROR | OMAudit | user=testuser2/scm@EXAMPLE.COM | ip=172.22.0.4 | op=RENEW_DELEGATION_TOKEN {kind=OzoneToken, service=172.22.0.9:9862} | ret=FAILURE | TOKEN_ERROR_OTHER ...
```